packages/tools/src/stateManagement/segmentation/config/segmentationVisibility.ts

```diff
@@ -15,10 +15,11 @@ function getSegmentationIndices(segmentationId) {
 
 /**
  * Set the visibility of a segmentation representation for a given tool group. It fires
- * a SEGMENTATION_REPRESENTATION_MODIFIED event. If the segmentation had some segments
- * hidden, it will remove them from the hidden set and make them all visible. On the
- * other hand, if the segmentation had all segments hidden, it will make all segments
- * visible.
+ * a SEGMENTATION_REPRESENTATION_MODIFIED event. If the visibility is set to true, and
```

Comment:
There is something perhaps I don't understand here, why do we mention things like "had some segments hidden/shown"... does this even have to be said? Is it not correct to say "visibility true will show all segments" and "visibility false will hide all segments"?

Reply:
ok changed it